test-packages/macro-fastboot-tests.test.js

```diff
@@ -0,0 +1,37 @@
+const execa = require('execa');
+const FastBoot = require('fastboot');
+
+test('FastBoot - macro-test app boots in FastBoot environment using custom app-boot', async () => {
+  jest.setTimeout(120000);
+
+  const buildProcess = execa('ember', ['build'], {
+    cwd: `${__dirname}/macro-tests`,
```

Comment:
This is fine for now, although I know ed would prefer us to create a fresh test-app for fastboot, but if you don't have cycles I can totally help.

Reply:
I created a separate fastboot-app for fastboot related testing.

Reply:
will be useful for any future changes related to Fastboot add-on